### PR TITLE
Fixes to exult packages (reopen)

### PR DESCRIPTION
**Pull request by winterheart**
Cleanup exult-sound and exult compilation options.

### CODE REVIEW

File: games-engines/exult/exult-9999.ebuild [hunk 1]
```diff
@@ -11,12 +11,14 @@ EGIT_REPO_URI="https://github.com/exult/exult"
...
 KEYWORDS=""
-IUSE="timidity zlib"
+IUSE="+sdl2 timidity zlib"
 
 DEPEND="
 	>=media-libs/libpng-1.2.43-r2:0
 	games-misc/exult-sound
-	media-libs/libsdl[sound,video,X]
+	media-libs/libvorbis
+	sdl2? ( media-libs/libsdl2[video,X] )
```
Comment (chewi) [inline games-engines/exult/exult-9999.ebuild]: No sound for libsdl2? May be technically correct, just seems odd.

File: games-engines/exult/exult-9999.ebuild [hunk 2]
```diff
@@ -35,13 +37,16 @@ src_prepare() {
...
+	local with_sdl="sdl12"
+	use sdl2 && with_sdl="sdl2"
 	econf \
 		--x-libraries="/usr/$(get_libdir)" \
 		--disable-tools \
 		--disable-opengl \
 		--enable-mods \
 		--with-desktopdir=/usr/share/applications \
 		--with-icondir=/usr/share/pixmaps \
+		--with-sdl=${with_sdl}
```
Comment (chewi) [inline games-engines/exult/exult-9999.ebuild]: Could simplify this with `--with-sdl=$(usex sdl2 sdl2 sdl12)`.

File: games-misc/exult-sound/exult-sound-0-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+EAPI=6
+
+DESCRIPTION="Sound data for games-engines/exult"
+HOMEPAGE="http://exult.sourceforge.net/"
+SRC_URI="mirror://sourceforge/exult/exult_audio.zip"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (chewi) [inline games-misc/exult-sound/exult-sound-0-r2.ebuild]: Empty IUSE can be left out entirely.

File: games-misc/exult-sound/exult-sound-0-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+DEPEND="!<games-engines/exult-9999"
+
+S="${WORKDIR}"
+
+src_install() {
+	insinto /usr/share/exult/music
+	doins "${WORKDIR}/music/"*ogg
```
Comment (chewi) [inline games-misc/exult-sound/exult-sound-0-r2.ebuild]: No need to reference WORKDIR all the time, just do `doins music/*.ogg` and likewise below.

**Comment by winterheart** [comment]
@chewi Done.

**Pull request closed without merging**